app/controllers/organized/admin_controller.rb

```diff
@@ -2,5 +2,6 @@ module Organized
   class AdminController < Organized::BaseController
     before_filter :ensure_admin!
     skip_before_filter :ensure_member!
+    before_filter :ensure_not_ambassador_organization!
```

Comment:
This isn't inherited from base controller?

Reply:
Ah thanks! That's vestigial.